test/support/commands.js

```diff
@@ -28,6 +28,23 @@ Cypress.Commands.add('drawRegion', ({ x = 200, y = 200, width = 100, height = 10
         .trigger('mouseup');
 });
 
+Cypress.Commands.add('selectText', ({ page = 0, block = 0 } = {}) => {
```

Comment:
Should page be defaulted to `1` and this method could handle the zero index shift to make it more clear?